lib/code_input.dart

```diff
@@ -47,7 +47,7 @@ class CodeInput extends StatefulWidget {
     TextInputType keyboardType = TextInputType.text,
     List<TextInputFormatter> inputFormatters,
     @required CodeInputBuilder builder,
-    EdgeInsetsGeometry padding = const EdgeInsets.only(left:2, right: 2),
+    double spacing = 2,
```

Comment:
`2` pixel feels a little small as a default, especially since before there was effectively a space of `4` pixel between characters. Maybe even change this to `8` pixel like in the example?
```suggestion
    double spacing = 8,
```